demos/serverless/src/handlers.js

```diff
@@ -3,7 +3,7 @@
 
 const AWS = require('./aws-sdk');
 const fs = require('fs');
-const uuid = require('./uuid/v4');
+const { v4: uuidv4 } = require('uuid');
```

Comment:
The deploy script is installing modules and copying `aws-sdk` and `uuid` packages to the local folder. I think you should keep loading it from locally but changing the left-hand only. Please try running the script in a fresh repo.

https://github.com/aws/amazon-chime-sdk-js/blob/a37d6d82208c79e79a44f8d9ed4c2e26539d8d3a/demos/serverless/deploy.js#L144-L148

Reply:
Updated: `const { v4: uuidv4 } = require('./uuid');`

Tested with both the videohelpdesk & serverless demo.